newsroom/core/resources/validators.py

```diff
@@ -1,7 +1,8 @@
 import re
 import ipaddress
 from typing import Any, Optional
-from pydantic import AfterValidator
+from bson import ObjectId, errors
```

Comment:
would we good to import only `ObjectId` and `InvalidId` from `bson` to avoid unnecessary imports and improve readability.

thoughts ?